{{cookiecutter.project_dashed}}-src/Pipfile

```diff
@@ -9,7 +9,8 @@ restview = "*"
 [packages]
 # Pin this until Keg fixes compatibility issues.
 flask = "<1.0"
-KegAuth = {git = "https://github.com/level12/keg-auth.git"}
+keyring = "*"
```

Comment:
keyring should only be a dev dependency.  It shouldn't be needed for tests or deployment.

Reply:
Done.